content/chainguard/migration-guides/migrating-to-php-images.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Migrating to PHP Chainguard Images"
+linktitle: "Migrating PHP"
+type: "article"
+description: "Guidance on how to migrate PHP Dockerfile workloads to use Chainguard Images"
+date: 2024-04-04T15:56:52-07:00
+lastmod: 2024-04-04T15:56:52-07:00
+draft: false
+tags: ["Images", "Product", "Conceptual"]
+images: []
+weight: 100
+toc: true
+---
+
+Chainguard Images are built on top of [Wolfi](/chainguard/open-source/wolfi/), a Linux _undistro_ designed specifically for containers. Our PHP images have a minimal design that ensures a smaller attack surface, which results in smaller images [with low to zero](/chainguard/chainguard-images/vuln-comparison/php/) CVEs. Nightly builds deliver fresh images whenever updated packages are available, which also helps to reduce the toil of manually patching CVEs in PHP images.
+
```

Comment:
One general note: there isn't a lot of consistency in whether it's stylized as "Image" or "image". I think for any instance where you're talking about one or more Chainguard images, you should capitalize the "I", and in cases where you're talking about images generally, it's fine to just leave it lowercase.

Reply:
Do you mean for instance "Our PHP Images" vs "Our PHP images"  ? The way I have been using it so far is whenever it's together "Chainguard Images" I make sure is uppercase, because it's the product name. But if I'm referring to the unit of image (one or more, seen as objects) then I am using the lowercase.